exonum/benches/criterion/transactions.rs

```diff
@@ -264,27 +230,21 @@ fn bench_verify_transactions_simple(b: &mut Bencher, &size: &usize) {
 
 fn bench_verify_transactions_event_loop(b: &mut Bencher, &size: &usize) {
     let transactions = gen_transactions(TRANSACTIONS_COUNT, size);
-    b.iter_with_large_setup(
-        || {
-            let dir = TempDir::new("exonum").unwrap();
-            let runner = TransactionsBenchmarkRunner::new(
-                create_rocksdb(&dir),
-                transactions
-                    .iter()
-                    .map(|transaction| transaction.raw().clone()),
-            );
-            (runner, dir)
-        },
-        |(runner, _dir)| {
-            runner.run().unwrap();
-        },
-    )
+
+    let verifier = TransactionVerifier::new();
+    let mut core = Core::new().unwrap();
+
+    b.iter(|| {
+        core.run(verifier.send_all(&transactions)).unwrap();
+    });
+    verifier.join();
 }
 
 pub fn bench_verify_transactions(c: &mut Criterion) {
     crypto::init();
 
-    let parameters = (7..12).map(|i| pow(2, i)).collect::<Vec<_>>();
+    let parameters = (7..12).map(|i| 1 << i).collect::<Vec<_>>();
```

Comment:
Yes, we know that previously in 90's bit shifting was faster than taking exponent of two, but can you revert your change in favour of readability

Reply:
This was me ✋  I think `1 << i` is no less readable than `pow(2, i)`, and I'm not even a C/C++ guy. Strictly personal opinion though; I won't insist on it.